README.md

```diff
@@ -143,6 +134,21 @@ before(:suite) { disable_production_minter! }
 after(:suite)  { enable_production_minter! }
 ```
 
+If you switch to the new database-backed minter and want to include in that minter the state of your current file-backed minter, AF::Noid 2.0.0 provides a new rake task that will copy your minter's state from the filesystem to the database:
+
+```bash
```

Comment:
No, we don't I suspect we can drop the **NOTE 1** content below in favor of the short-and-sweet block up here.